lib/elixir/src/elixir_import.erl

```diff
@@ -12,8 +12,14 @@ import(Meta, Ref, Opts, E) ->
         {Added1, Funs} = import_functions(Meta, Ref, Opts, E),
         {Funs, keydelete(Ref, ?key(E, macros)), Added1};
       {only, macros} ->
-        {Added2, Macs} = import_macros(true, Meta, Ref, Opts, E),
-        {keydelete(Ref, ?key(E, functions)), Macs, Added2};
+        {Added1, Macs} = import_macros(true, Meta, Ref, Opts, E),
+        {keydelete(Ref, ?key(E, functions)), Macs, Added1};
+      {only, sigils} ->
+        {Added1, Funs1} = import_functions(Meta, Ref, Opts, E),
+        {Added2, Macs1} = import_macros(false, Meta, Ref, Opts, E),
+        Funs2 = keyupdate(Ref, fun take_sigils/1, Funs1),
+        Macs2 = keyupdate(Ref, fun take_sigils/1, Macs1),
+        {Funs2, Macs2, Added1 or Added2};
```

Comment:
I believe the trace will return `imported: true` even if no sigils are imported, and I think the trace in that case (if no sigils are imported) should be `imported: false`. In other words, I think `Added1 or Added2` needs to be calculated after filtering the sigils.